tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "baseUrl": ".",
     "esModuleInterop": true,
     "jsx": "react",
-    "lib": ["dom", "esnext"],
+    "lib": ["esnext", "dom"],
```

Comment:
TS was yelling about `Object.entries` not existing, and I think it had to do with `dom` being the first target for `lib`. We want to target `esnext` to get access to all thee latest features.